0032-peering-clearing-settlement/0032-peering-clearing-settlement.md

```diff
@@ -0,0 +1,160 @@
+
+Nonetheless, it is likely that future bilateral protocols (like BTP) will define more detailed processes for reconciliation and clearing.
+
+# Settlement
+
+It is unlikely that the flow of packets between two peers will always net out such that the balance continuously tends toward zero. In most cases flows will be predominantly in one direction and there will be a need for the peers to reduce them back to 0.
+
+This avoids a situation where one peer owes the other a large sum of money and is then unable or unwilling to pay (settlement risk). Connectors should define a maximum Interledger account balance to manage the settlement risk they expose themselves to with their peers.
+
+Further, the principal of the Interledger network is to settle as often as is practically possible and financially viable such that this risk is very low.
+
+In the case of peers that use payment channels to settle this can be done for every packet as the cost of settlement is almost zero (the cost of exchanging claims on the channel). This reduces the settlement risk between peers down to the value of only those packets that are not yet fulfilled, rejected or expired (i.e. packets that are in-flight).
+
+Connectors will configure their own business rules regarding when to settle, based on how much they trust their peers and the costs and speed of settlement on the underlying network.
+
+For most implementations, this configuration will consist of, at a minimum, a maximum balance, and a settlement threshold. 
+
+## Maximum Balance
+
+When a connector receives an incoming packet from a peer it will ensure that the amount of the packet, added to the current balance, does not exceed the maximum balance. If it does it MUST reject he packet with a T04 error.
+
+This is only likely to occur if: 
+ 1. The settlement threshold is lower than the additive inverse (negation) of the maximum balance at the peer, or
+ 1. The sending peer is unable to settle the account fast enough to keep up with the total amount in the packets being sent.
+
+In the former case the peer may choose to perform a settlement, even though it has not reached its settlement threshold so that the balance on the Interledger account at the peer is reduced below the maximum. 
+
+In the latter case, the sending peer may need to throttle back on the packets it sends to the upstream peer or they will need to make an alternatiuve settlement arrangement that can accomodate the volume.
+
+## Settlement Threshold
+
+When a connector receives a successful response from a peer it will evaluate if the amount of the packet, subtracted from the current balance, is less than the settlement threshold. If so, the connector should perform a settlement, following which the balance on the Interledger account is adjusted up by the settlement amount.
```

Comment:
This is a bit hard to follow so it might be good to add `(see below for an example)`